### PR TITLE
feat(vue): add scrollOnChange option to prevent scrolling to top

### PR DESCRIPTION
**Pull request by jessevanmuijden**
When users change the filter options and fewer hits are found than before changing the filter, the page jumps to the top. It should be possible to disable this automatic scrolling behavior. This has been implemented in the React version of the library some time ago (see https://github.com/appbaseio/reactivesearch/issues/528).

### CODE REVIEW

File: packages/vue/src/components/result/ReactiveList.jsx
Comment (jyash97) [inline packages/vue/src/components/result/ReactiveList.jsx]: Hey thanks for the PR 💯 

default value should be `true` like it is in React version

**Review by jyash97 — APPROVED** [review]
Thanks @jessevanmuijden 

Looks good to me ✨  Waiting for another approval & we can merge it than.

**Comment by jyash97** [comment]
@jessevanmuijden  You have missed this one place -> https://github.com/appbaseio/reactivesearch/blob/82c08cbdce0653a376dba5507313f8edefa654ae/packages/vue/src/components/result/ReactiveList.jsx#L269

You may have to do the changes here as well.

**Review by bietkul — APPROVED** [review]

**Comment by bietkul** [comment]
@jessevanmuijden Thanks for the PR, it'll be out in next release.

**Pull request merged**
